transformers/uws/server.js

```diff
@@ -69,6 +69,8 @@ module.exports = function server() {
     spark.emit('incoming::data', msg);
   });
 
+  native.server.group.onPing(group, (msg, spark) => spark.heartbeat());
```

Comment:
So the change to `incoming::ping` means that the handler is properly removed on end and there's not a chance of calling `spark.heartbeat()` on a closed spark. Makes sense.